projects/novo-elements/src/elements/field/formats/date-time-format.ts

```diff
@@ -160,7 +160,7 @@ export class NovoDateTimeFormatDirective extends IMaskDirective<any> implements
     const dateTime =  text.split(', ');
     const hour: string = dateTime[1].slice(0, 2);
     if (!this.military) {
-      const input = dateTime[1].substr(17, 4).replace(/\-/g, '').trim().slice(0, 2);
+      const input = dateTime[1].substr(5, 4).replace(/\-/g, '').trim().slice(0, 2);
```

Comment:
what does this change represent? was this previously broken?

Reply:
This was a bug that was found from when I made the date-time formatter. I was working off the full string not the split string.